### PR TITLE
Version 07

### PR DESCRIPTION
**Pull request by wtoorop**
- Spelling fixes from Tim Wicinski and Josh Soref
- Replace SHOULDs with MUSTs for ignoring things that are meaningless to a catalog consumer
- Update the list of people to thank in the Acknowledgements section
- Mention PowerDNS 4.7.0 support for catalog zones

### CODE REVIEW

**Review by peterthomassen — APPROVED** [review]
lgtm, thanks!

**Review by libor-peltan-cznic — APPROVED** [review]

File: draft-ietf-dnsop-dns-catalog-zones.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ defined in [@!RFC1982].  Otherwise, catalog consumers might not notice
 updates to the catalog zone's contents.
 
 There is no requirement to be able to query the catalog zone via recursive nameservers.
-Catalog consumers SHOULD ignore NS record at apex.
+Catalog consumers MUST ignore NS record at apex.
```
Comment (mind04) [inline draft-ietf-dnsop-dns-catalog-zones.md:179]:
```suggestion
Catalog consumers MUST ignore the absence of NS record at apex.
```

File: draft-ietf-dnsop-dns-catalog-zones.md [hunk 2]
```diff
@@ -336,7 +336,7 @@ Implementations and operators of catalog zones may choose to provide their own p
 Custom properties can occur both globally, or for a specific member zone.
 To prevent a name clash with future properties, such properties should be represented below the label `ext`.
 
-`ext` is not a placeholder, so a custom property would have domains names as follows:
+`ext` is not a placeholder, so a custom property would have domain names as follows:
```
Comment (mind04) [inline draft-ietf-dnsop-dns-catalog-zones.md:339]: I'm not sure 'domain' is appropriate here.

File: draft-ietf-dnsop-dns-catalog-zones.md (hunk 1 above)
Comment (Habbie) [reply draft-ietf-dnsop-dns-catalog-zones.md:179]: (although I suspect BIND might currently not accept that)

Comment (klaus-nicat) [reply draft-ietf-dnsop-dns-catalog-zones.md:179]: Maybe NS records can be used to send implicit NOTIFYs to downstream catalog consumers? Although explicit also-notifys are probably more useful in such scenarios.

File: draft-ietf-dnsop-dns-catalog-zones.md (hunk 2 above)
Comment (wtoorop) [inline draft-ietf-dnsop-dns-catalog-zones.md:339]:
```suggestion
`ext` is not a placeholder. A custom property is named as follows:
```

@mind04 Is this better?

File: draft-ietf-dnsop-dns-catalog-zones.md [hunk 3]
```diff
@@ -516,7 +516,7 @@ Thanks to Leo Vandewoestijne. Leo's presentation in the DNS devroom at the
 FOSDEM'20 [@FOSDEM20] was one of the motivations to take up and continue the
 effort of standardizing catalog zones.
 
-Thanks to Brian Conry, Klaus Darilion, Brian Dickson, Tony Finch, Evan Hunt, Shane Kerr, Patrik Lundin, Matthijs Mekking, Victoria Risk, Petr Spacek and Carsten Strotmann for reviewing draft proposals and offering comments and suggestions.
+Thanks to Brian Conry, Klaus Darilion, Brian Dickson, Tony Finch, Evan Hunt, Shane Kerr, Patrik Lundin, Matthijs Mekking, Victoria Risk, Petr Spacek, Carsten Strotmann, Josh Soref, Michael StJohns and Tim Wicinski for reviewing draft proposals and offering comments and suggestions.
```
Comment (wtoorop) [inline draft-ietf-dnsop-dns-catalog-zones.md:519]:
```suggestion
Thanks to Brian Conry, Klaus Darilion, Brian Dickson, Tony Finch, Evan Hunt, Shane Kerr, Patrik Lundin, Matthijs Mekking, Victoria Risk, Josh Soref, Petr Spacek, Michael StJohns, Carsten Strotmann and Tim Wicinski for reviewing draft proposals and offering comments and suggestions.
```
Reorder surname

File: draft-ietf-dnsop-dns-catalog-zones.md [hunk 4]
```diff
@@ -176,7 +176,7 @@ defined in [@!RFC1982].  Otherwise, catalog consumers might not notice
 updates to the catalog zone's contents.
 
 There is no requirement to be able to query the catalog zone via recursive nameservers.
-Catalog consumers SHOULD ignore NS record at apex.
+Catalog consumers MUST ignore NS record at apex.
 However, at least one is still required so that catalog zones are syntactically correct DNS zones.
 A single NS RR with a NSDNAME field containing the absolute name "invalid." is RECOMMENDED [@!RFC2606;@!RFC6761].
```
Comment (wtoorop) [inline draft-ietf-dnsop-dns-catalog-zones.md:181]:
```suggestion
Therefore NS RRs at the apex are meaningless to the catalog consumer and MUST like any other meaningless RRs be ignored.
However, to accommodate name server software that refuses to load zones without an NS RRset at a zone's apex, we RECOMMEND to have a single NS RR with a NSNAME field  containing the absolute name "invalid." [@!RFC2606;@!RFC6761]
```

File: draft-ietf-dnsop-dns-catalog-zones.md (hunk 1 above)
Comment (Habbie) [reply draft-ietf-dnsop-dns-catalog-zones.md:179]: The text should definitely allow an implementation to use the NS records for that.

File: draft-ietf-dnsop-dns-catalog-zones.md (hunk 4 above)
Comment (Habbie) [reply draft-ietf-dnsop-dns-catalog-zones.md:181]: Ah, but #53 fixed this by simplifying the requirement to a central "must ignore whatever they don't care about", so I think we can resolve this conversation.

Comment (wtoorop) [reply draft-ietf-dnsop-dns-catalog-zones.md:181]: @Habbie Yes, I now also prefer removing the sentence actually. So I prefer PR #53  and would like to merge it sooner than later, but #53 itself does still have a nit I'd like your opinion on.

File: draft-ietf-dnsop-dns-catalog-zones.md (hunk 1 above)
Comment (wtoorop) [reply draft-ietf-dnsop-dns-catalog-zones.md:179]: I closed this conversation in favor of PR #53

File: draft-ietf-dnsop-dns-catalog-zones.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (wtoorop) [reply draft-ietf-dnsop-dns-catalog-zones.md]: This has been fixed in a suggestion.

File: draft-ietf-dnsop-dns-catalog-zones.md (hunk 2 above)
Comment (wtoorop) [reply draft-ietf-dnsop-dns-catalog-zones.md:339]: @mind04 See the suggestion to resolve this below

Comment (mind04) [reply draft-ietf-dnsop-dns-catalog-zones.md:339]: Yes, much better

**Review by mind04 — APPROVED** [review]

File: draft-ietf-dnsop-dns-catalog-zones.md (hunk 2 above)
Comment (wtoorop) [reply draft-ietf-dnsop-dns-catalog-zones.md]: Resolved by the suggestion

**Pull request merged**